e2e/bitrise.yml

```diff
@@ -46,10 +45,74 @@ app:
       </plist>
 
 workflows:
+  test_auto_codesign_with_manual_assets:
+    before_run:
+    - _setup
+    steps:
+    - certificate-and-profile-installer:
+        inputs:
+        - certificate_url: $BITFALL_APPLE_IOS_CERTIFICATE_URL_LIST
+        - certificate_passphrase: $BITFALL_APPLE_IOS_CERTIFICATE_PASSPHRASE_LIST
+        - provisioning_profile_url: $BITFALL_APPLE_PROVISIONING_PROFILE_URL_LIST
+        - install_defaults: "no"
+    - path::./:
+        title: Step Test - Multi Target API Key auto signing
+        inputs:
+        - distribution_method: development
+        - archive_path: ./archives/Fruta.xcarchive
+        - product: app
+        - automatic_code_signing: api-key
+        - min_profile_validity: 30
+        - certificate_url_list: $BITFALL_APPLE_IOS_CERTIFICATE_URL_LIST
+        - passphrase_list: $BITFALL_APPLE_IOS_CERTIFICATE_PASSPHRASE_LIST
+
+  test_auto_codesign:
+    before_run:
+    - _setup
+    steps:
+    - path::./:
+        title: Step Test - Multi Target xcodebuild auto signing
+        run_if: |-
+          {{ enveq "IS_LATEST_STACK_XCODE" "true" }}
+        inputs:
+        - distribution_method: development
+        - archive_path: ./archives/Fruta-managed.xcarchive
+        - product: app
```

Comment:
Is app-clip also supported?

Reply:
To some extent: 
- both xcodebuild and bitrise-managed (appstoreconnect based) automatic signing method fails, if the App Identifier does not exist for the App Clip on the developer portal. 
- xcodebuild can enable missing capabilities, while bitrise-managed
- if app id is in sync with the project (code managed and appstoreconnect based) are fine
- apple id version is not tested yet